### ISSUE TITLE
Erroneous statements about MIDI Note Off Velocity

### ISSUE THREAD

**Issue by PeteCA** [csound/manual#171]
There are several places in the manual that mention using the Velocity value of a MIDI Note Off message.  They are all completely wrong!

The midinoteoff entry seems to indicate that it returns the key number and velocity of a Note Off.  It doesn't.  It's always triggered at Note *On*, and returns the values of that event.  The page should probably be removed unless/until it can be made to work.

The pages on the "r" events all reference note-off velocities.  The 'linenr' page, for instance, states:

   "These “r” units can also be modified by MIDI noteoff velocities (see veloffs)."

There is no indication of how they might be so modified (and in fact, they can't), and there is no opcode "veloffs" -- and never has been.

Those statements, or slight variants, have been in the manual for around twenty years, which shows, I suppose, the true importance of Note Off Velocities to users!

**Comment by ReinerZorn** [comment]
I agree that note-off velocity is important. I wrote an instrument using this with Supercollider and it allows an intuitive control of the release of a sound. Of course it is not aftertouch but at least a bit like this. Therefore, I strongly encourage that `midinoteoff` is implemented as it is expected to work.

For the moment I am trying a workaround based on this:

```kstatus, kchan, kdata1, kdata2 midiin
kstatus, kchan, kdata1, kdata2 midiin
knewmidi changed kstatus
if knewmidi==1 then
  if kstatus==128 then
    printk 0, kdata2
  endif
endif
```

This reads the note-off event as a generic MIDI event and prints the note-off velocity. In my tests it did not interfere with the note-on because that is already read from MIDI when the instrument is started from MIDI. As soon as the status changes it reads the value which is the note-off velocity. Of course, it can happen that accidentally the pitch bend or mod wheel was changed. So in addition it has to be checked that the status changed to 128 and not to something else.

I am not yet sure whether this interferes with intended use of pitch bend and mod wheel with standard commands as pchbend in instruments using these.
